infra/toynet/terraform/devUsers.tfvars

```diff
@@ -1,5 +1,5 @@
-instance_names = ["toynet-tay"]
+instance_names = ["toynet-theo"]
 instance_count = 1
 
-ami_id = "ami-0ae6b743a09efa056"
-key_name = "toynet-2020"
+ami_id = "ami-0303d20c69955b4b5"
```

Comment:
Did you make a new AMI using packer?

Reply:
Yes, this "ami_id" came from the `packer build toynet.json` command. I have a screenshot if you want it.